examples/typescript/evm/account.signTypedData.ts

```diff
@@ -0,0 +1,49 @@
+// Usage: pnpm tsx evm/account.signTypedData.ts
+
+import { CdpClient } from "@coinbase/cdp-sdk";
+import "dotenv/config";
+
+const cdp = new CdpClient();
+
+const account = await cdp.evm.createAccount();
+
+console.log("Created account:", account.address);
+
+const signature = await account.signTypedData({
+  message: {
```

Comment:
Let's remove this top-level `message` grouping, as it's part of the OpenAPI spec and adds a little bit of unnecessary friction here